contracts/math/README.md

```diff
@@ -0,0 +1,49 @@
+# DecimalMath
+
+This is an Ethereum library to implement decimal arithmetic tailored to cryptocurrency operations.
+
+## Usage
+Tokens implementing `ERC20Detailed` already are in a decimal representation, with a virtual comma `decimals()` positions to the left of the least significant digit.
+
+Using tokens and wei, with `N` decimals, the number stored to represent 1 wei would be 1, and the number stored to represent 1 token would be 100...`pad(0, N)`...0 (== 10**decimals()).
+
+`DecimalMath` implements the four basic arithmetic operations (`addd`, `subd`, `muld` and `divd`) to operate on token amounts respecting the position of the comma.
+
+`DecimalMath` assumes that all operands are in the widespread `wei` representation.
+
+When using `DecimalMath` the representation should be the same. For example:
+ - Multiply 1 token by 0.5: `DecimalMath.muld(1000000000000000000, 500000000000000000)`
+ - Multiply 1 wei by 2: `DecimalMath.muld(1, 2000000000000000000)`
+ - Multiply 1.01 by 2.02: `DecimalMath.muld(1010000000000000000, 2020000000000000000)`
+
+## Number Conversions
+
+No functions are supplied to convert to and from decimal representation, as an user you just need to remember that you need to add 18 zeros (or multiply by `token.decimals().unit()`) to convert from solidity integer to DecimalMath decimal. Remove zeros or divide by `token.decimlas().unit()` to convert back.
```

Comment:
Wouldn't it be `DecimalMath.unit()`?

Reply:
`unit(uint8 decimals)` is the function at hand. One would add `using DecimalMath for uint8` and then do token.decimals().unit().

Reply:
Ah, I see. You are right, but it is a bit confusing to do it that way. I would use the more verbose `DecimalMath.unit(token.decimals())` in the README so that it is clear what is going on.